packages/editor/src/components/rich-text/index.js

```diff
@@ -151,10 +151,10 @@ export class RichText extends Component {
 	 * @return {Object} The current record (value and selection).
 	 */
 	getRecord() {
-		const { formats, text } = this.formatToValue( this.props.value );
+		const { formats, lineFormats, objects, text } = this.formatToValue( this.props.value );
```

Comment:
The introduction of `lineFormats` and `objects` makes the role or meaning of `formats` unclear. Should `formats` be renamed to something like `textFormats`, `characterFormats`, `spanFormats` or whatever makes sense?

Reply:
I guess anything else named formats should be avoided. I'd like to keep `formats` for inline formats and the maybe change `lineFormats` to `lines` or something like that?

Reply:
`lines` could be good, yeah.